src/ai/backend/agent/stats.py

```diff
@@ -525,10 +526,14 @@ async def collect_per_container_process_stat(
             updated_cids: Set[ContainerId] = set()
             for result in results:
                 if isinstance(result, Exception):
-                    log.error(
-                        "collect_per_container_process_stat(): gather_process_measures() error",
-                        exc_info=result,
-                    )
+                    if type(result) == psutil.NoSuchProcess:    
+                        log.debug(f"{result}")
+                        pass
+                    else:
+                        log.error(
+                            "collect_per_container_process_stat(): gather_process_measures() error",
+                            exc_info=result,
+                        )
```

Comment:
I think we can use pattern matching here!
Let's try it.
```python
match result:
    case psutil.NoSuchProcess():
        log.debug(str(result))
    case Exception():
        log.error()
        ...
    case _:
        log.error()
        ...
```

Reply:
Applied it right away 🚀

Reply:
Just FYI: This kind of `asyncio.gather(..., return_exceptions=True)` pattern will be replaced with [`aiotools.as_completed_safe()`](https://aiotools.readthedocs.io/en/latest/aiotools.utils.html#aiotools.utils.as_completed_safe) in the future.

For now, Python's pattern matching syntax would be a good fit here.